spec/javascript/components/lease_ups/__snapshots__/LeaseUpsPage.test.js.snap

```diff
@@ -1279,610 +1279,7634 @@ Array [
                 </div>
               </div>
             </div>
-          </div>
```

Comment:
@akegan are you able to speak to all the changes in this snapshot?

Reply:
Sure! I haven't gone over them line-by-line but I think they are all mostly empty rows being added since now we're loading a table with 20 rows instead of 5 rows.

Reply:
yes looks like that sort of thing mostly. i'm wondering why the "previous" button is removed here? we can also look at this when i stop by the DS office later